### ISSUE TITLE
Config examples - don't use variables

### ISSUE THREAD

**Issue by lauracowen** [OpenLiberty/docs#1877]
Use actual example directories instead of variables in the config examples. I know the SMEs like to give variables but these config examples are meant to be _examples_ - not generic templates that contain variables for the reader to resolve in their heads. eg
```
<library id="jCacheVendorLib">
    <fileset dir="${server.config.dir}/hazelcast" includes="hazelcast.jar"/>
  </library>
  <httpSessionCache libraryRef="jCacheVendorLib"
                    uri="file:${server.config.dir}/hazelcast/hazelcast.xml"/>
```
Where it says `${server.config.dir}`, it looks more complicated than it needs to be. And it’d be easy to just say `uri="file:server1/hazelcast/hazelcast.xml"`
(I think I’m right in thinking that that variable resolves to the server’s directory, which is the same as the server name?).

Similarly for `${wlp.install.dir}`. just say `wlp/` instead. As that’s correct for the majority of users.

I think it’s okay to use the variables _when necessary_ in other topics (but mostly just use the real paths instead of variables - the variables are for the computer to resolve in the Liberty runtime, not for people to read).

However, we need to be using the values consistently. eg `server1` say for the config dir. Don't vary this between different examples/topics unless it's for illustrating a point.

BTW, we’re unlikely to have more than one server in these examples anyway because if we’re primarily talking containers, there’d only be one server per container. Even if someone isn’t using containers, a single-server configuration is the simplest and they can adapt it to more complicated situations from there.

**Comment by dmuelle** [comment]
The following features use `${server.config.dir}` in config examples & need to be updated to use `server1` instead:

- [x] audit

- [x] jdbc

- [x] ldapRegistry

- [x] sessionCache

- [x] sessionDatabase

- [x] socialLogin

- [x] spnego

- [x] transportSecurity

**Comment by dmuelle** [comment]
fix for published docs delivered to vNext in #2361 

fixes to draft docs delivered by writers owning the issues and confirmed
